recipes/openssl/1.x.x/conanfile.py

```diff
@@ -436,9 +436,15 @@ def _get_env_build(self):
             self._env_build = AutoToolsBuildEnvironment(self)
         return self._env_build
 
+    def _get_default_openssl_dir(self):
+        if self.settings.os == "Linux" and self._full_version >= "1.1.0":
+            return "/etc/ssl"
+        return os.path.join(self.package_folder, "res")
+
     @property
     def _configure_args(self):
-        openssldir = self.options.openssldir if self.options.openssldir else os.path.join(self.package_folder, "res")
+        defaultopenssldir = self._get_default_openssl_dir()
+        openssldir = self.options.openssldir if self.options.openssldir else defaultopenssldir
```

Comment:
```suggestion
        openssldir = self.options.openssldir or self._get_default_openssl_dir()
```

I think this should work and it's easier to read (and more pythonic)

Reply:
Nice, thanks :)